zebrad/src/commands.rs

```diff
@@ -23,6 +23,11 @@ use std::path::PathBuf;
 /// Zebrad Configuration Filename
 pub const CONFIG_FILE: &str = "zebrad.toml";
 
+/// Controls how far ahead of the chain tip the syncer tries to download before
+/// waiting for queued verifications to complete. Set to twice the maximum
+/// checkpoint distance.
+pub const LOOKAHEAD_LIMIT: usize = 2 * 2_000;
+
```

Comment:
This constant already exists at:
https://github.com/ZcashFoundation/zebra/blob/77a1fefa1ee54485729509ab0f566fbf0d7c7de0/zebrad/src/commands/start/sync.rs#L23